__tests__/diff.test.ts

```diff
@@ -345,7 +345,7 @@ describe('summary function', () => {
       )})) 🔒`
     )
     expect(result).toContain(
-      `This comment was last updated at 1/1/2023 12:00:00 PM`
+      `This comment was last updated at Sun, 01 Jan 2023 12:00:00 GMT`
```

Comment:
In `diff.ts`, the date format was changed to use `.toUTCString()`. It explicitly shows the timezone (previously it was the "server timezone", which was ambiguous). Also, `01 Jan` is easier to interpret than `1/1`. :)